src/main/java/org/spongepowered/api/entity/living/Human.java

```diff
@@ -71,6 +71,52 @@
     void setSaturation(float saturation);
 
     /**
+     * Gets the current experience towards the next level.
+     *
+     * <p>This is not the total experience this human has.</p>
+     *
+     * @return The current experience towards the next level
+     */
+    int getExperience();
+
+    /**
+     * Gets the current experience level of this human.
+     *
+     * @return The current level
+     */
+    int getLevel();
+
+    /**
+     * Gets the total accumulated experience this human has, including
+     * the experience to achieve the current experience level and the
+     * experienced gained towards the next level.
+     *
+     * @return The current total accumulated experience
+     */
+    int getTotalExperinece();
+
+    /**
+     * Sets the experience accumulated towards the next level.
```

Comment:
There should be a note about how much experience advances a player to the next level.